indigo_app/static/javascript/indigo/views/work.js

```diff
@@ -346,15 +346,15 @@
     },
   });
 
-  Indigo.BulkImportWorksView = Backbone.View.extend({
+  Indigo.BatchAddWorkView = Backbone.View.extend({
```

Comment:
not sure why but I'm no longer seeing a progress bar

Reply:
hard refresh?